### DISCUSSION TITLE
Git Shows Changes between header lines

### DISCUSSION THREAD

**Discussion by ccantrell72** [fdbozzo/foxbin2prg#102] [Q&A]
These two lines are coming up always changed between the updates of bin>txt. I am not seeing how they are different.

* (ES) AUTOGENERADO - ��ATENCI�N!! - ��NO PENSADO PARA EJECUTAR!! USAR SOLAMENTE PARA INTEGRAR CAMBIOS Y ALMACENAR CON HERRAMIENTAS SCM!!
* (ES) AUTOGENERADO - ��ATENCI�N!! - ��NO PENSADO PARA EJECUTAR!! USAR SOLAMENTE PARA INTEGRAR CAMBIOS Y ALMACENAR CON HERRAMIENTAS SCM!!

**Comment by lscheffler** [comment]
@ccantrell72    
Again, I need the version number.

Not with me.   
Can you open the old and the new version in an HEX editor of your choice and check for the difference? (Just cut off the rest) or use a compare  tool that can do this? The strings at github tend to be altered by github.

In special check for LF and CRLF.

**Reply by ccantrell72** [reply]
1.20.078666

**Reply by ccantrell72** [reply]
I'll take a look at using another tool to see.

**Reply by ccantrell72** [reply]
Yes, it does look to have changed in hex.

<img width="904" alt="Pasted Graphic" src="https://github.com/fdbozzo/foxbin2prg/assets/104120598/7f383e3d-7ce3-49e4-92bc-afd20ba1c454">

**Reply by ccantrell72** [reply]
Thanks!!!

**Reply by lscheffler** [reply]
can you point me to the change?

**Reply by lscheffler** [reply]
This is what FoxBin2Prg writes into the file
![grafik](https://github.com/fdbozzo/foxbin2prg/assets/27769909/6cc0eef6-d00d-43f9-9363-4f2c4a764f0f)
from this code
```
`		TEXT TO lcText ADDITIVE TEXTMERGE NOSHOW FLAGS 1 PRETEXT 1+2
			*--------------------------------------------------------------------------------------------------------------------------------------------------------
			* (ES) AUTOGENERADO - ¡¡ATENCIÓN!! - ¡¡NO PENSADO PARA EJECUTAR!! USAR SOLAMENTE PARA INTEGRAR CAMBIOS Y ALMACENAR CON HERRAMIENTAS SCM!!
			* (EN) AUTOGENERATED - ATTENTION!! - NOT INTENDED FOR EXECUTION!! USE ONLY FOR MERGING CHANGES AND STORING WITH SCM TOOLS!!
			*-------------------------------------------------------------------------------------------------------------------------------------------------------
```

I've checked my *.??2 files, and they come with bytes as written by FoxBin2Prg.

So the problem seems to be, that something changes the HI-ASCII stuff.   Do you have any idea what process is doing that?

I personally think we should remove the line at all, even if if ends up rewriting any text file.

@all, what do you think about this?

**Comment by DougHennig** [comment]
I agree: they should be removed.

**Comment by lscheffler** [comment]
Fixed in 1.21.0
